### PR TITLE
Add metamorphic "Blood" sprite

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

If blood is such a delicious drink and people drink it off the floor all the time.. why isn't there a metamorphic sprite of it? This addition will make you feel like a vampire!

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image
image

## Breaking changes

Right now there are a couple errors which not only don't change the metamorphic glass sprite, and other weird stuff that just doesn't work.

image
image

**Changelog**
:cl:
- add: Blood now has a sprite when placed in metamorphic glasses.

### CODE REVIEW

File: Resources/Prototypes/Reagents/biological.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
-﻿- type: reagent
```
Comment (deltanedas) [inline Resources/Prototypes/Reagents/biological.yml:1]: sus

File: Resources/Textures/Objects/Consumable/Drinks/bloodglass.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "Drawn by Ubaser", "states": [{"name": "icon"}]}
```
Comment (deltanedas) [inline Resources/Textures/Objects/Consumable/Drinks/bloodglass.rsi/meta.json:1]: formatting :trollface:

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Reagents/biological.yml (hunk 1 above)
Comment (UbaserB) [reply Resources/Prototypes/Reagents/biological.yml:1]: What's wrong with it? When i compare it to other reagents, it looks normal
image

Comment (deltanedas) [reply Resources/Prototypes/Reagents/biological.yml:1]: the diff is sus

**Comment by UbaserB** [comment]
wut

**Comment by UbaserB** [comment]
yknow thats stupid this time i never even tapped force push

File: Resources/Textures/Objects/Consumable/Drinks/bloodglass.rsi/meta.json (hunk 2 above)
Comment (UbaserB) [reply Resources/Textures/Objects/Consumable/Drinks/bloodglass.rsi/meta.json:1]: just checked, its very standard for drink formatting? Its the same so im going to set it to resolved.

File: Resources/Prototypes/Reagents/biological.yml (hunk 1 above)
Comment (UbaserB) [reply Resources/Prototypes/Reagents/biological.yml:1]: ok i literally have no idea how to fix this but i doubt its what is causing such a big error

**Pull request merged**